### DISCUSSION TITLE
table树形数据展示,无法控制是否行展开

### DISCUSSION THREAD

**Discussion by rabbitbiubiubiu** [ant-design/ant-design#31721] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
根据用户的条件,判定是否展开本行

### What does the proposed API look like?
table树形数据展示时,是否展开行可控

**Comment by yissan** [comment]
我也是同样的问题，现在控制是否能支持展开必须要依赖于expandedRowRender，rowExpandable貌似没有啥用
